test/resources/custom_scenario_template/docker/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/INSTALL.rst

```diff
@@ -0,0 +1,22 @@
+*******
```

Comment:
Maybe let's remove this file since it is in the testing and not user facing?

Reply:
removed